### PR TITLE
Restyle the services/machines switcher

### PR DESCRIPTION
**Pull request by anthonydillon**

### CODE REVIEW

**Comment by anthonydillon** [comment]
## QA
- Load the GUI and check the service/machines switcher looks like the design [1]
- Make sure they work as expected

[1] https://raw.githubusercontent.com/CanonicalLtd/juju-design/master/Nav/GUI/GUI_Masthead.png?token=ABWRnld9i2xD3cbOjAXtZuyss13szfpRks5XIy9NwA%3D%3D

**Comment by jujugui** [comment]
Test FAILed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/4904/
Test FAILed.

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/4905/
Test PASSed.

File: jujugui/static/gui/src/app/assets/svgs/changes-machine-created.svg [hunk 1]
```diff
@@ -1,123 +1,5 @@
...
-           x="240"
-           height="1"
-           width="15.999992"
-           id="rect4277"
-           style="color:#000000;display:inline;overflow:visible;visibility:visible;fill:#38b44a;fill-opacity:1;fill-rule:nonzero;stroke:none;stroke-width:2;marker:none;enable-background:accumulate" />
-      </g>
-    </g>
-  </g>
+418 bytes - 77.15% saving
+<svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" viewBox="0 0 16.000017 16.000017">
```
Comment (hatched) [inline jujugui/static/gui/src/app/assets/svgs/changes-machine-created.svg]: Just so you're aware the build process runs all of the svg's through svg-sprite shrinking them so you don't need to pre-shrink.

**Comment by hatched** [comment]
Looks good! Thanks :+1: :ok:

**Comment by makyo** [comment]
:+1: and :shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci.jujugui.org:8080/job/juju-gui-merge

**Comment by anthonydillon** [comment]
@hatched understand the builds creates SVG sprite but I needed to rewrite the SVG to work with styling the path as there was a full background path which was turning the icon into an orange square. So was easier to refactor with a tidied up SVG.

**Pull request merged**